src/iexamine.cpp

```diff
@@ -2558,6 +2558,11 @@ void iexamine::dirtmound( Character &you, const tripoint &examp )
     }
     const auto &seed_id = std::get<0>( seed_entries[seed_index] );
 
+    if( !here.has_flag_ter_or_furn( seed_id->seed->required_terrain_flag, examp ) ) {
+        add_msg( _( "This type of seed can not be planted in this location." ) );
```

Comment:
_( str ) marks str for extraction, and will return the translated string for the set language, so this is all you need to do to ensure translation is done.

Reply:
Thanks!